### ISSUE TITLE
FAILED: FileAndResourceDirectivesSpec getFromResource_should_reject_requests_to_directory_resources_from_an_archive

### ISSUE THREAD

**Issue by jrudolph** [akka/akka-http#1560]
https://jenkins.akka.io:8498/view/Akka-http/job/akka-http-nightly-against-akka-master/362/testReport/akka.http.scaladsl.server.directives/FileAndResourceDirectivesSpec/getFromResource_should_reject_requests_to_directory_resources_from_an_archive/

```
Error Message

true did not equal false


Stacktrace

org.scalatest.exceptions.TestFailedException: true did not equal false
	at org.scalatest.MatchersHelper$.indicateFailure(MatchersHelper.scala:340)
	at org.scalatest.Matchers$AnyShouldWrapper.shouldEqual(Matchers.scala:6742)
	at akka.http.scaladsl.server.directives.FileAndResourceDirectivesSpec$$anonfun$3$$anonfun$apply$mcV$sp$22$$anonfun$apply$24.apply(FileAndResourceDirectivesSpec.scala:211)
	at akka.http.scaladsl.server.directives.FileAndResourceDirectivesSpec$$anonfun$3$$anonfun$apply$mcV$sp$22$$anonfun$apply$24.apply(FileAndResourceDirectivesSpec.scala:211)
	at scala.util.DynamicVariable.withValue(DynamicVariable.scala:58)
	at akka.http.scaladsl.testkit.RouteTest$$anonfun$check$1.apply(RouteTest.scala:57)
	at akka.http.scaladsl.testkit.RouteTest$$anonfun$check$1.apply(RouteTest.scala:57)
	at akka.http.scaladsl.testkit.RouteTestResultComponent$RouteTestResult.$tilde$greater(RouteTestResultComponent.scala:50)
	at akka.http.scaladsl.server.directives.FileAndResourceDirectivesSpec$$anonfun$3$$anonfun$apply$mcV$sp$22.apply(FileAndResourceDirectivesSpec.scala:211)
	at akka.http.scaladsl.server.directives.FileAndResourceDirectivesSpec$$anonfun$3$$anonfun$apply$mcV$sp$22.apply(FileAndResourceDirectivesSpec.scala:211)
```

**Comment by jrudolph** [comment]
https://jenkins.akka.io:8498/job/pr-validator-akka-http/1763/testReport/akka.http.scaladsl.server.directives/FileAndResourceDirectivesSpec/getFromResource_should_reject_requests_to_directory_resources_from_an_archive/

**Comment by jrudolph** [comment]
Hopefully fixed by #1565.
